plugins/woocommerce/src/Internal/Admin/Orders/ListTable.php

```diff
@@ -4,6 +4,7 @@
 
 use Automattic\WooCommerce\Internal\DataStores\Orders\CustomOrdersTableController;
 use Automattic\WooCommerce\Internal\DataStores\Orders\OrdersTableDataStore;
+use Automattic\WooCommerce\RestApi\UnitTests\Helpers\ProductHelper;
```

Comment:
This import doesn't seem to be required.

Reply:
Good catch, thank you! Addressed in [`37151703`](https://github.com/woocommerce/woocommerce/pull/33789/commits/37151703714a16ab95526c51e048256e264b799e).